src/Compilers/Core/Portable/InternalUtilities/EnumUtilties.cs

```diff
@@ -16,7 +16,7 @@ internal static class EnumUtilities
         /// </summary>
         /// <remarks>
         /// </remarks>
-        internal static ulong ConvertEnumUnderlyingTypeToUInt64(object value, SpecialType specialType)
+        internal static ulong ConvertEnumUnderlyingTypeToUInt64(object? value, SpecialType specialType)
```

Comment:
Should the parameter be `object` instead? We're asserting `value != null` below.

Reply:
Quite right.